lib/index.js

```diff
@@ -678,8 +678,24 @@ var Client = module.exports = function(config) {
             console.log("REQUEST: ", options);
 
         function httpSendRequest() {
-            var reqModule = self.config.followRedirects === false ? protocol : 'follow-redirects/' + protocol;
```

Comment:
Yes - Browserify/webpack both analyze the require paths statically, so they both can't follow paths constructed at runtime.